packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+// No restrictions for using iframe :)
+// functions and parameters such as autoplay, playVideo(), loadVideoById() won't work in all mobile environments
+
+class YoutubePlayer extends Component {
+
+  constructor(props) {
+    super(props);
+
+    const { selectedTrack, id } = props
+
+    this.state = {
+      playerAnchor: React.createRef(),
+      selectedTrack,
+      id
+    }
+
+    // use static methods?
+    this.loadAPI = this.loadAPI.bind(this);
```

Comment:
This can be a class method instead, since it has no reliance on the class instance.